data-esnext.js

```diff
@@ -4144,6 +4144,36 @@ exports.tests = [
     },
   ]
 },
+{
+  name: 'Hashbang Grammar',
+  category: STAGE3,
+  significance: 'tiny',
+  spec: 'https://github.com/tc39/proposal-hashbang/',
+  mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Hashbang_comments',
+  exec: function() {/*
+    try {
+      return !eval('#!/wash/your/hands');
+    } catch (e) {
+      return false
+    }
+  */},
+  res: {
+    chrome1: false,
+    chrome74: true,
+    firefox2: false,
+    firefox67: true,
+    ie11: false,
+    opera10_50: false,
+    edge18: false,
```

Comment:
Could you please add `edge79: true`?

Reply:
I think `edge78+` maps to `chrome 78+` if support data is missing. So `chrome74: true` should implies `edge78: true`. If it turns out that it doesn't work on `edge 78`, then we should mark as unsupported, otherwise I am good to leave it as-is.

Reply:
oh yes indeed :)